pandas/core/indexes/period.py

```diff
@@ -270,9 +423,46 @@ def __contains__(self, key):
     def _int64index(self):
         return Int64Index._simple_new(self.asi8, name=self.name)
 
-    @property
-    def values(self):
-        return self.astype(object).values
+    # ------------------------------------------------------------------------
+    # Index Methods
+
+    @deprecate_kwarg(old_arg_name='n', new_arg_name='periods')
+    def shift(self, periods):
```

Comment:
can this share a doc-string with DTI?

Reply:
That's the wrong shift :), what we call `_time_shift` internally now. We want the `Series.shift` kind here.